GPU/Common/PostShader.cpp

```diff
@@ -216,6 +224,8 @@ void LoadPostShaderInfo(Draw::DrawContext *draw, const std::vector<Path> &direct
 					if (info.scaleFactor >= 2 && info.scaleFactor < 8) {
 						appendTextureShader(info);
 					}
+				} else if (!section.name().empty()) {
+					WARN_LOG(G3D, "Unrecognized shader type '%s' or invalid shader in section '%s' : %d", shaderType.c_str(), section.name().c_str());
```

Comment:
Oops, format error... not sure what the %d was?

-[Unknown]

Reply:
I had things organized slightly differently here first, but forget what the %d was for. Getting rid of it, thanks for spotting.